pnpm-lock.yaml

```diff
@@ -1,4 +1,8 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Comment:
I think the pnpm version needs to be updated in the GitHub workflows for this to work.